src/H5Mpublic.h

```diff
@@ -245,9 +245,27 @@ H5_DLL hid_t  H5Mcreate_async(hid_t loc_id, const char *name, hid_t key_type_id,
 /**
  * \ingroup H5M
  *
- * \brief
+ * \brief Creates a map object without linking it into a file
  *
- * \details
+ * \fgdta_loc_id
+ * \type_id{key_type_id}
+ * \type_id{val_type_id}
+ * \mcpl_id
+ * \mapl_id
+ * \return \hid_t{map object}
+ *          The resulting ID should be linked into the file with H5Olink or it
+ *          will be deleted when closed.
+ *
+ * \details H5Mcreate_anon() creates a new map object for storing key-value
+ *          pairs. The in-file datatype for keys is defined by \p key_type_id
+ *          and the in-file datatype for values is defined by \p val_type_id. \p
+ *          loc_id specifies the file to create the map object, but no link to
+ *          the object is created.  Other options can be specified through the
+ *          property lists \p mcpl_id and \p mapl_id.
+ *
+ *          The new map should be linked into the group hierarchy before being
+ *          closed or it will be deleted. The dataset should be closed when the
```

Comment:
I got this from the source code but wasn't sure about the second sentence.  Please advice.